build/nuget/uno.winui.common.targets

```diff
@@ -186,37 +186,6 @@
              ContinueOnError="true" />
 	</Target>
 
-	<Target Name="ValidateUnoUIAndroid" BeforeTargets="BeforeBuild" Condition="'$(AndroidApplication)'!='' and '$(TargetFrameworkVersion)'!='' and '$(_AndroidApiLevel)'!=''">
-
-		<PropertyGroup Condition="'$(NETCoreAppMaximumVersion)'!='' and '$(NETCoreAppMaximumVersion)'&gt;='6.0'">
-			<UnoUIMinAndroidSDKVersion>31</UnoUIMinAndroidSDKVersion>
-			<_CurrentTrimmedAndroidSDLVersion>$(_AndroidApiLevel)</_CurrentTrimmedAndroidSDLVersion>
-		</PropertyGroup>
-
-		<!-- JELA TODO -->
-		<PropertyGroup Condition="$(TargetFramework.ToLowerInvariant().StartsWith('monoandroid')) or '$(ProjectTypeGuids)'=='{EFBA0AD7-5A72-4C68-AF49-83D382785DCF};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}'">
-			<UnoUIMinAndroidSDKVersion>12.0</UnoUIMinAndroidSDKVersion>
-			<_CurrentTrimmedAndroidSDLVersion>$(TargetFrameworkVersion.Substring(1))</_CurrentTrimmedAndroidSDLVersion>
-		</PropertyGroup>
-
-		<Error Text="This version of the Android SDK ($(_CurrentTrimmedAndroidSDLVersion)) is not supported by Uno Platform. You must change the &quot;Compile using Android version:&quot; field in the android project property with at least version $(UnoUIMinAndroidSDKVersion)."
-					 Condition="'$(_CurrentTrimmedAndroidSDLVersion)' &lt; '$(UnoUIMinAndroidSDKVersion)'" />
-	</Target>
-
-	<!-- Validation for https://github.com/unoplatform/uno/issues/9430 and the static registrar requirement -->
-	<Target Name="_ValidateIOSStaticRegistrar"
-			BeforeTargets="BeforeBuild"
-			Condition="
-			'$(UnoDisableValidateIOSStaticRegistrar)'=='' and
-			'$(TargetFramework)'=='net6.0-ios' and '$(RuntimeIdentifier)'=='iossimulator-x64' and (
```

Comment:
This is now always false as the whole package no longer supports net6.0, and the issue on Xamarin side is already fixed

Reply:
lgtm